random1on1/api/channels.py

```diff
@@ -1,36 +1,132 @@
+import json
 from abc import ABC
 from abc import abstractmethod
+from datetime import datetime
+from functools import reduce
 
-from discord.channel import GroupChannel
-from discord.channel import TextChannel
+from discord import CategoryChannel
+from discord import GroupChannel
+from discord import Role
+from discord import TextChannel
+from networkx import connected_components
+from networkx import union
+from preconditions import preconditions
 
-from random1on1.api.participant import Participant
+from .pairings import Pairings
```

Comment:
Yeah I tried, but pyright language server complained -- if there's a good way to avoid them and satisfy the error handling for the language servers happy to do that

Reply:
Which, what, server are you talking about? Can you give me a reference for what's the problem?